lxd/device/unix_hotplug.go

```diff
@@ -149,29 +148,36 @@ func (d *unixHotplug) Start() (*deviceConfig.RunConfig, error) {
 	runConf := deviceConfig.RunConfig{}
 	runConf.PostHooks = []func() error{d.Register}
 
-	device := d.loadUnixDevice()
-	if d.isRequired() && device == nil {
+	devices := d.loadUnixDevices()
+	if d.isRequired() && devices == nil {
```

Comment:
Can we do `if d.isRequired() && len(devices) <= 0 {` to catch the scenario where devices is non-nil but zero length?

Reply:
Sure thing 👍